pkg/process/net/common_windows.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	connectionsURL = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/connections"
-	registerURL    = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/register"
-	statsURL       = "http://localhost:3333/debug/stats"
-	netType        = "tcp"
+	connectionsURL       = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/connections"
+	registerURL          = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/register"
+	languageDetectionURL = "http://unix/" + string(sysconfig.LanguageDetectionModule) + "/detect"
```

Comment:
is it supported on windows? the go detection is using the `debug/elf` package

Reply:
It is not. I added it so that `pkg/process/net/common.go` doesn't have any build errors. We make sure that it isn't called here https://github.com/DataDog/datadog-agent/blob/5c102e2bb2b1af849d1381071527fcb6654e4493/pkg/languagedetection/cli_detector.go#L172. I can adjust the url to match the others however